Paws4Me/Paws4Me/Controllers/Logic/SignInViewController.swift

```diff
@@ -0,0 +1,142 @@
+//
+//  SignInViewController.swift
+//  Paws4Me
+//
+//  Created by Kirsty-Lee Walker on 2022/02/26.
+//
+import UIKit
+
+protocol Validation {
+    func isValidCredentialsUsername(username: String) -> Bool
+    func isValidCredentialsPassword(password: String) -> Bool
+}
+
+let bottomLine = CALayer()
+let bottomLine2 = CALayer()
+
+class SignInViewController: UIViewController, Validation {
+    func isValidCredentialsUsername(username: String) -> Bool {
+        if username == "Admin"{
+            return true
+        } else {
+            return false
+        }
+    }
+
+    func isValidCredentialsPassword(password: String) -> Bool {
+        if password == "TestPass123" {
+            return true
+        } else {
+            return false
+        }
+    }
+    @IBOutlet weak var txtUsername: UITextField!
+    @IBOutlet weak var txtPassword: UITextField!
+    @IBOutlet weak var btnSignIn: UIButton!
+    @IBOutlet weak var btnRegister: UIButton!
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        addCornerRadius(btnSignIn)
+        addCornerRadius(btnRegister)
+        changeBorderLook(btnRegister)
+        addUnderline(txtUsername, txtPassword, bottomLine: bottomLine, bottomLine2: bottomLine2)
+    }
+
+    @IBAction private func btnSignInClick(_ sender: UIButton!) {
+        guard let username = txtUsername.text else {return}
+        guard let password = txtPassword.text else {return}
+
+        let bValidUsername = isValidCredentialsUsername(username: username)
```

Comment:
So no real need to store these values, you could just call the function in the actual if check 😄 also I don't recommend using this naming convention 👍